xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -126,9 +126,10 @@
         <ReturnType>System.IO.Stream</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a reference to the underlying stream.</summary>
+        <value>A stream object that represents the underlying stream.</value>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ObjectDisposedException">The underlying stream is closed.</exception> <!-- Question for reviewers: The actual exception message is 'Cannot access a closed file.'. Should I use that string instead? The string 'The underlying stream is closed' is used in GZipStream and DeflateStream and explains 'when' the exception is thrown, but is not the actual exception message. -->
```

Comment:
You don't have to use the exact same error message. The docs should explain when you get that exception.

Reply:
Thank you. I removed the comment with the question.